src/main.js

```diff
@@ -0,0 +1,66 @@
+var suggested;
+
+var letsCookButton = document.querySelector(".lets-cook-button")
+var side = document.querySelector(".side");
+var mainDish = document.querySelector(".main-dish");
+var dessert = document.querySelector(".dessert");
+var entireMeal = document.querySelector(".entire-meal");
+var cookpot = document.querySelector(".cookpot");
+var foodSuggestion = document.querySelector(".food-suggestion");
+var suggestion = document.querySelector(".suggestion");
+// var clearButton = document.querySelector(".clear-button");
```

Comment:
I like leaving notes for future iterations. I don't know if this is acceptable practice when merging into main?